### PR TITLE
command

### PR DESCRIPTION
**Pull request by GitAYYYer**

### CODE REVIEW

**Review by BrodeyYendall — APPROVED** [review]
LGTM 👍

**Comment by GitAYYYer** [comment]
LGTM 👍

File: commands/currency-conversion/currencyConversion.test.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      isCommand: jest.fn(() => true),
+      commandName: "convert-aud-to-jpy",
+      reply: jest.fn(),
+    };
+  });
+
+  it("replies with the conversion rate when API call is successful", async () => {
+    const mockApiResponse = {
+      conversion_rates: {
+        JPY: 80.5,
```
Comment (BrodeyYendall) [inline commands/currency-conversion/currencyConversion.test.js:20]: the conversion rate hasnt been this low in 3 years, this test in inaccurate.

**Review by BrodeyYendall — CHANGES_REQUESTED** [review]
please update tests with valid data

**Comment by DylanSbogar** [comment]
Lgtm

**Pull request merged**